scrapi/base/schemas.py

```diff
@@ -27,8 +30,8 @@
     "uris": {
         "canonicalUri": ('//dc:identifier/node()', oai_extract_url)
```

Comment:
Lots of times identifiers return multiple results, should we add the "single_result" here too?